modules/syntax/syntax.h

```diff
@@ -44,26 +44,9 @@ typedef struct hlcolour {
 } hlcolour;
 
 // here is an array of syntax highlights by extensions, keywords, comments, del
-struct editor_syntax highlight_db[] = {
-  CCPP_syntax,
-  CSHARP_syntax,
-  Python_syntax,
-  PHP_syntax,
-  PAS_syntax,
-  SQL_syntax,
-  JAVA_syntax,
-  JAVASCRIPT_syntax,
-  GO_syntax,
-  RUBY_syntax,
-  RUST_syntax,
-  HTML_syntax,
-  D_syntax,
-  BRAIN_syntax,
-  BASH_syntax,
-  MAKEFILE_syntax
-};
-
-#define HIGHLIGHT_DB_ENTRIES (sizeof(highlight_db)/sizeof(highlight_db[0]))
```

Comment:
I can see you have moved lots of definitions in C source file, I hope you have not moved those which are common to multiple source file. Let's say for example if in future it is required to use `struct editor_syntax highlight_db[]` to define a variable in one of the source file then one needs to declare the struct as `extern` in that source file.
I would like suggestions from others as well on this. @shnupta what do you think on this?